nsqadmin/http.go

```diff
@@ -732,6 +732,21 @@ func (s *httpServer) doConfig(w http.ResponseWriter, req *http.Request, ps httpr
 			if err != nil {
 				return nil, http_api.Err{400, "INVALID_VALUE"}
 			}
+		case "log_level":
+			var logLevelStr string // temporary variable in case the value is invalid
+			err := json.Unmarshal(body, &logLevelStr)
```

Comment:
I was testing this PR locally and ran into this - I don't think this should be json, it should just be a plain string.

Treating the body as json representation of a single string means you need to include the quotes e.g. `"info"` instead of `info` and that seems a bit silly.

(Same comment applies for nsqd.)